setup.py

```diff
@@ -32,7 +32,7 @@ def readme():
     license="MIT",
     packages=["Plugwise_Smile"],
     install_requires=[
-        "aiohttp",
+        "asyncio",
```

Comment:
#86 says:

Found conflicts:
- asyncio
- datetime

Why remove aiohttp and add asyncio?

Reply:
Yeah, mixup 👎